README.md

```diff
@@ -1,5 +1,13 @@
 # Project manager API
 
+## Setup instructions
+
+This API runs on the iO Academy docker infrastructure (see [the docker-image repository](https://github.com/iO-Academy/docker-image/)).
+
+1. Run `composer dump-autoload`
+2. Import `db/project_manager.sql` into MariaDB
```

Comment:
need to change to mysql and add instruction on what to name the new db (as discussed)

Reply:
done!